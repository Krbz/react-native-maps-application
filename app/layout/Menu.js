import React, { Component } from "react";
import { connect } from "react-redux";
import { styles } from "../styles.android";

import { ScrollView, Text } from "react-native";

export class MenuLayout extends Component {
  constructor(props) {
    super(props);

    this.state = {
      drawer: {
        visible: true,
      }
    };

    this.closeDrawer = this.closeDrawer.bind(this);
  }

  closeDrawer() {
    this.props.drawerActions.hideDrawer(this.state.drawer);
  }

  render() {
    return (
      <ScrollView style={styles.AppDrawer}>
        <Text>Menu</Text>
      </ScrollView>
    );
  }
}

/* istanbul ignore next */
function mapStateToProps(state) {
  return {

  }
}

/* istanbul ignore next */
function mapDispatchToProps(dispatch) {
  return {

  };
}

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(MenuLayout)
