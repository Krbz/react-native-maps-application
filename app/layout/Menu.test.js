import "react-native";
import React from "react";
import { MenuLayout } from "./Menu";

import renderer from "react-test-renderer";

test('renders correctly', () => {
  const tree = renderer.create(
    <MenuLayout />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});