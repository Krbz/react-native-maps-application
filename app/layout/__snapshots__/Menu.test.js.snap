// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView
  style={
    Object {
      "backgroundColor": "#eeeeee",
    }
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
    >
      Menu
    </Text>
  </View>
</RCTScrollView>
`;
